Sources/DocCDocumentation/DocCDocumentation.docc/V5_API_Migration_guide.md

```diff
@@ -39,9 +39,7 @@ If you're using the Firebase SDK, you'll need to follow [these instructions](htt
 Version 5.0 of the SDK introduces support for observer mode when making purchases with StoreKit 2. You can enable it when configuring the SDK:
 
 ```swift
-Purchases.configure(with: .builder(withAPIKey: apiKey)
-		.with(observerMode: true, storeKitVersion: .storeKit2)
-    .build()
+Purchases.configure(withAPIKey: apiKey, observerMode: true, storeKitVersion: .storeKit2)
```

Comment:
We don't currently have a configure function with this signature - currently if you want to use observerMode with this type of configure function you need to provide an appUserID. Perhaps we should add a configure function with this signature in the future, but for now this won't build.

FWIW, you can configure observer mode without an appUserID, but you need to use the builder pattern:
```swift
Purchases.configure(with: .builder(withAPIKey: apiKey)
         .with(observerMode: true, storeKitVersion: .storeKit2)
         .build()
     )
```

```suggestion
Purchases.configure(withAPIKey: apiKey, appUserID: appUserID, observerMode: true, storeKitVersion: .storeKit2)
```